src/interactableComponents/drawer/index.js

```diff
@@ -3,11 +3,15 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import {StyleSheet, Animated, View, TouchableOpacity, TouchableHighlight} from 'react-native';
 import Interactable from 'react-native-interactable';
-import {BaseComponent} from '../../commons';
+// import {BaseComponent} from '../../commons';
+import baseComponent from './../../commons/baseComponent';
 import {screenWidth} from '../../helpers/Constants';
 import {Colors, Typography} from '../../style';
 import NewDrawer from './newDrawer';
 
+
+const BaseComponent = baseComponent(true);
```

Comment:
you don't need to do that (: 
you can just import `PureBaseComponent` from ` '../../commons'`

Reply:
Ok. Done.